### ISSUE TITLE
Licensing for this project is unclear

### ISSUE THREAD

**Issue by Mr0grog** [CDCgov/prime-reportstream#270]
**What type of help do you need?**

* [x] Question
* [ ] New Repo
* [ ] Delete Repo
* [ ] Other

**Please describe how you'd like us to help.**

While reading through the docs in this repo, I found myself a bit confused as to how it is licensed:
- The [`LICENSE` file in the repo root](https://github.com/CDCgov/prime-data-hub/blob/3ed4ea665b16e3073a9d313ac3e744b8537be257/LICENSE) is the Apache 2 License.
- [`CONTRIBUTING.md`](https://github.com/CDCgov/prime-data-hub/blob/3ed4ea665b16e3073a9d313ac3e744b8537be257/CONTRIBUTING.md#public-domain) says the project is in the public domain.
- [`README.md`](https://github.com/CDCgov/prime-data-hub/blob/3ed4ea665b16e3073a9d313ac3e744b8537be257/README.md#public-domain-standard-notice) lists the project as being in the public domain *and* being licensed via Apache 2 in sections right next to each other, and does not clarify that differing circumstances apply to each, as in other dual-licensed open-source projects (e.g. public domain within the United States and Apache 2 elsewhere).

This might leave potential users and contributors both confused about the particular terms and rights that apply to this project, since these two licensing schemes (CC0/public domain vs. Apache 2) definitely conflict with each other. Which is correct, or is there a more nuanced explanation, like certain files are licensed one way while others are licensed differently, or different jurisdictions of use where one or the other license applies?

**Comment by leebrian** [comment]
FYI, I noticed this issue and am interested in what you develop for your project and needs. I set up the template with both because creative comments is meant to address copyright and apache is meant to address software license. That doesn't mean they conflict with each other, but they aren't incompatible by nature, and wanted to have something operational as most programs don't customize this (although I wish they would).

Apache includes a section for copyright, but since we make government products that are in the public domain we don't typically copyright anything we create. So the intent is to note that the material in the project is available under creative commons in terms of IP usage. And that the material is available under an apache license that includes lots more rights and restrictions than copyright. At one point we didn't include a license since just being under public domain means it can be used in pretty much every way that ASL grants, but it was harder on users who were trying to evaluate license compatibility for their projects. So Apache was added to make projects more compatible so by default material are available under both CC0 and ASL. Of course, programs can adjust this to meet particular needs.

Still confusing, but I hope this context helps, and always looking for ways to make this wording better so that we get better contributions and more usage.
